### DISCUSSION TITLE
Lspsoed parasitic manager not able to open it through shortcut for the second time

### DISCUSSION THREAD

**Discussion by ghost** [mywalkb/LSPosed_mod#79] [Q&A]
When I fresh install the module I am only able to open the parasitic manager for the first time, when I close it or after rebooting the Shortcut is there but it doesn't open the parasitic manager. If I disabled the parasitic notification, I need to install the lsposed manager app again to be able to access it. Attached the screenshot of what happens when I click on the shortcut and I am using the latest canary Lsposed build.
![Screenshot_Shell](https://github.com/mywalkb/LSPosed_mod/assets/124941348/56e163c5-77ed-403a-b6a8-b7044672b0a7)

**Comment by mywalkb** [comment]
can you provide logs and more details about your device?

**Reply by mywalkb** [reply]
I see only io.mesalabs.knoxpatch as installed module, have you tried to install other modules except knoxpatch? If yes have you the same behaviour?
Without modules installed everything works normally right?
It seems that your device is not compatible with the knoxpatch module at least that's what it looks like from the log.

**Reply by ghost** [reply]
I have the same behavior even installing other modules and even without any modules it still appears like that. Knox patch is compatible with my device as the Knox apps work and when I click on the app info of the shortcut the app name is called Shell.

**Reply by mywalkb** [reply]
Try to disable all modules and catch logs after the popup error, no reboot, if you can take logs from installed manager is good.
In your logs I can't see error except for module knoxpatch.

**Reply by mywalkb** [reply]
Have you issues with installed manager?
Have you tried original LSPosed?
Have you tried magisk?
When you install the modules do they continue to work after the manager starts giving you errors?

**Reply by mywalkb** [reply]
> I just needed to give root access to shell 

strange, usually I have root permissions to the shell, but even if I remove them the shortcut continues to work.
Parasitic manager works with hooks.

**Reply by ghost** [reply]
Stopped working again 😒 
![Screenshot_093650_Shell](https://github.com/mywalkb/LSPosed_mod/assets/124941348/313e6ca3-6047-46d0-9b38-19bcefd3d8dd)

**Reply by ghost** [reply]
I don't have issues with the installed manager but it recommends me to use the parasitic manager, I get the same message with the original Lspsoed before it wasn't like this. The modules still work for me even after these error and I didn't try with Magisk.

**Reply by mywalkb** [reply]
> before it wasn't like this

Do you know what event triggered this issue?
android update, lsposed update, kernelsu update or others

**Reply by ghost** [reply]
Actually now it is working, the issue appeared after I went to the Canary verison. To clear memory the phone stops the background running apps so the Shell apparently should be running in the background for the shortcut to work as it was closed the  shortcut didn't work when I added it to excluded apps list to not close it now it is working perfectly.

**Reply by elswerky** [reply]
> > I just needed to give root access to shell
> 
> strange, usually I have root permissions to the shell, but even if I remove them the shortcut continues to work. Parasitic manager works with hooks.

 Ro 
This issue occur with last few months updates of oneui Samsung devices 

And not all devices 

Samsung changed something as I expect  ,I hope you can figure it And fix it

**Reply by elswerky** [reply]
> > before it wasn't like this
> 
> Do you know what event triggered this issue? android update, lsposed update, kernelsu update or others

Samsung update in last about 6 months ,, some Samsung devices have this issues,, others not affected
But it is related to an update in oneui 5.1. , I have note 10 lite on ineui 5.1 without issue ,so some devices only

**Comment by elswerky** [comment]
Foe example I faced this issue on tab S6 lite ,, but when I revert back to old update with same binary I was able to get this issue fixed 


So it is related to something changed by Samsung in last few updates. , and again nig all Samsung devices gave this issue with last updates
